### ISSUE TITLE
Issue with Notify by Email

### ISSUE THREAD

**Issue by killmasta93** [frappe/erpnext#17649]
Hi,
I was wondering if someone could assist me, currently the email alerts are working fine for calendar or reset password. but when i share a task it shows the box at the bottom that says notify by email but i get not email. But im sure the email alerts works because when i rest password alerts and also i get notified by email by calendar alerts. any ideas?
 currently on ERPNext: v11.1.21 (master)
Thank you

**Comment by clarkejj** [comment]
Refer to this https://discuss.erpnext.com/t/issue-with-notification-email/48900

**Comment by killmasta93** [comment]
yes correct i posted it there which to see if someone else has had the same issue and it seems that no one has it so posted it as a bug but then realized that sharing the task i get no email alert but assigning it i do get the email alert so there is something odd
